src/tests/ftest/util/scrubber_test_base.py

```diff
@@ -131,3 +132,40 @@ def run_ior_and_check_scruber_status(self, pool, cont, fail_on_warning=True):
         # Just make sure scrubber is working here.
         status = self.verify_scrubber_metrics_value(self.initial_metrics, self.final_metrics)
         return status
+
+    def run_mdtest_and_check_scruber_status(self, pool, cont, mdtest_params):
+        """Run mdtest and get scrubber metrics
+
+        Args:
+            pool (object): Pool object
+            cont (object): Container object within the pool.
+            mdtest_params(list): List comprising of different set of mdtest parameters.
+
+        Returns:
+            status(bool) : True (Scrubber working), False(Scrubber not working)
+        """
+        status = False
+        self.initial_metrics = self.scrubber.get_csum_total_metrics()
+        self.pool = pool
+        self.container = cont
+        # Print the pool properties
+        result = self.dmg_cmd.pool_get_prop(self.pool.uuid, "scrub")
+        self.log.info("Pool Properties")
+        self.log.info("===============")
+        self.log.info(result)
+        result = self.daos_cmd.container_get_prop(self.pool.uuid, self.container.uuid)
+        self.log.info("Container Properties")
+        self.log.info("===============")
+        self.log.info(result)
+
+        # Add a thread for these mdtest run
+        process = threading.Thread(target=self.run_mdtest_multiple_variants,
+                                   kwargs={"mdtest_params": mdtest_params})
+        # Launch the mdtest thread
+        process.start()
+        # Wait for the thread to finish
+        process.join()
+        self.final_metrics = self.scrubber.get_csum_total_metrics()
+        # Just make sure scrubber is working here.
+        status = self.verify_scrubber_metrics_value(self.initial_metrics, self.final_metrics)
```

Comment:
I wonder why you need to verify the metrics here because you verify it in your test (Line 47 of check_csum_metrics_mdtest.py).

Reply:
Agree... I am removing the new method from the scrubber_test_base.py and performing the changes only on the test script alone.